packages/golden_toolkit/test/screen_matches_golden_test.dart

```diff
@@ -3,23 +3,36 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:golden_toolkit/src/testing_tools.dart';
 
 void main() {
-  testGoldens('Animation test with CustomPump', (tester) async {
+  testGoldens('Custom Pump should be evaluated if supplied', (tester) async {
```

Comment:
this test started failing on Flutter Master because something changed with CircularProgressIndicator.

The circular progress indicator was an implementation detail of the test that distracted from the actual test case. Reworked the test to better demonstrate how customPump (btw, I HATE that name...) can be used for fine grained control of pumping... this should be less brittle. Verified it passes successfully on both stable & master